source/common/http/conn_manager_impl.cc

```diff
@@ -806,6 +806,32 @@ void ConnectionManagerImpl::ActiveStream::decodeData(Buffer::Instance& data, boo
   decodeData(nullptr, data, end_stream);
 }
 
+bool ConnectionManagerImpl::ActiveStream::seenEndStream(ActiveStreamDecoderFilter* filter) {
```

Comment:
Sorry, what's the reasoning for doing this change? What you had before seems more clear to me?

Reply:
OK,  I reverted the late change, and update the comment.